layers/+tools/shell/packages.el

```diff
@@ -28,11 +28,13 @@
         (term :location built-in)
         xterm-color
         vi-tilde-fringe
+        (vterm :toggle (not (eq window-system 'w32)))
```

Comment:
Use `spacemacs/system-is-mswindows`, which consults `system-type` instead of `window-system`.

Reply:
Should I also make the change where I found the example I based myself off of, in [shell/config.el](https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Btools/shell/config.el#L22) ?

Reply:
yes please